main/main.tex

```diff
@@ -54,113 +62,110 @@
 are of low quality, such as those with a high number of missing values, 
 ahead-of-time termination of chromatographic runs, or low instrument sensitivity. 
-Following the definitions from \cite{Bittremieux2017}, \texttt{MsQuality}
+Following the definitions by\cite{Bittremieux2017}, \texttt{MsQuality}
 focuses on the calculation of inter-experiment metrics, which can typically be 
-obtained by summation from an intra-experiment metric. Examples for
+obtained by summation \fixme{'can typically' needed here? 'summation' or 'summarization'?}  from an intra-experiment metric. Examples for
 intra-experiment metrics are the chromatogram of the total ion current (TIC) 
 over the retention time. Inter-experiment metrics, on the other hand, 
 facilitate the comparison of multiple MS runs or experiments, 
-e.g. via longitudinal analysis of quality metrics, such as the
+e.g., via longitudinal analysis of quality metrics, such as the
 fractions of the total retention time required to accumulate a given
 percentile of the TIC.
 
 \section{Usage scenario and implementation} \label{usagescenario}
 
-\texttt{MsQuality} offers easy-to-use means of evaluating data quality on a 
-per-measurement basis, including the identification of low-quality measurements 
-(e.g. poor intensities, low signal-to-noise ratio), biases and outliers, 
-variations in calibration, and batch and confounding effects within datasets
-(Fig. \ref{fig:fig1} a and b). 
-Utilizing community standards for data representation in mass spectrometry 
-defined by HUPO-PSI, \texttt{MsQuality}'s metric calculations enable 
-data quality comparison, storage, and facile exchange.
+\texttt{MsQuality} offers easy-to-use means of evaluating data quality on a
+per-measurement basis, including the identification of low-quality measurements (e.g. poor
+intensities, low signal-to-noise ratio), \fixme{some repetition of Intro here, trim}
+biases and outliers, variations in calibration, and batch and confounding effects within
+datasets (Fig. \ref{fig:fig1} a and b).  Its use of community standards for data
+representation in mass spectrometry defined by HUPO-PSI facilitates comparison, consistent
+storage, reporting and exchange of quality metrics and quality control criteria.
 
 \begin{figure}[ht!]
     \centering
- 	\includegraphics[scale=0.421]{/g/huber/users/naake/GitHub/MsQuality_manuscript/main/figure-main}
+ 	\includegraphics[scale=0.421]{figure-main}
  	  \caption{Examples of \texttt{MsQuality} functionality. Metrics are based
- 	        on MS1 spectra and one data point is obtained per MS1 spectra.
- 	        (a) Area under TIC: The area under the total ion chromatogram. 
-            (b) Area under TIC RT quantiles: The area under the total ion
-                chromatogram of the retention time quantiles. 
+ 	        on MS1 spectra; one data point is obtained per MS1 spectra\fixme{spectrum?}.
+ 	        (a) Area under TIC: The area under the total ion chromatogram. \fixme{$x$-axis is ugly (10e3)}
```

Comment:
Do you have any suggestions to improve this without showing too many digits, @wolfganghuber ?